pkg/cmd/cmd.go

```diff
@@ -83,12 +94,41 @@ func newFullCoverageCommand() *cobra.Command {
 					float64(info.TotalCoveredLines)/float64(info.TotalLines)*100,
 				)
 			}
+
+			if kustoOption.UseKusto {
+				dbClient, err := dbclient.NewKustoClient(kustoOption)
+				if err != nil {
+					return fmt.Errorf("new kusto client: %w", err)
+				}
+
+				now := time.Now().UTC()
+				for _, info := range all {
+					err = dbClient.Store(context.Background(), &dbclient.Data{
+						PreciseTimestamp: now,
+						LinesCovered:     info.TotalCoveredLines,
+						LinesValid:       info.TotalLines,
+						ModulePath:       moduleHostPath,
+						FilePath:         info.Path,
+						Coverage:         float64(info.TotalCoveredLines) / float64(info.TotalLines),
+					})
+					if err != nil {
+						return fmt.Errorf("store data: %w", err)
+					}
+				}
+
+			}
+
 			return nil
 		},
 	}
 
 	cmd.Flags().StringVar(&coverProfile, "cover-profile", "", `coverage profile produced by 'go test'`)
 	cmd.Flags().StringVar(&moduleHostPath, "host-path", "", "host path for the go project")
+	cmd.Flags().BoolVar(&kustoOption.UseKusto, "use-kusto", false, "whether enable kusto to collect coverage data")
```

Comment:
@dyu1208 please help review the option names.

Reply:
can we change it to "collect-coverage-data" ？

Reply:
I think it should convey that it's a bool flag.

Reply:
"coverage-data-collection-enabled" or shorten it to "data-collection-enabled"

Reply:
But if we want to support other db backend, such as mysql, nosql ?